docs/releases/versions/version-v11.0.0.md

```diff
@@ -0,0 +1,125 @@
+
+## What's new?
+
+TODO
+
+## Changes
+
+### Breaking changes
+
+- TODO
+
+### Minor changes
+
+#### Docker images and helm charts
+
+TODO
+
+#### Override deposit form template
+
+The deposit form template can now be overriden by editing the configuration variable `APP_RDM_DEPOSIT_FORM_TEMPLATE`.
+
+#### CLI commands
+
+##### Rebuild all search indices
+
+A new command `rebuild-all-indices` was added to `invenio rdm` command. It will rebuild the index of every service that is registered in the service registry.
+
+##### Confirm user on creation
+
+Command `invenio users create` has a new flag `--confirm`, or `-c` in short, that automatically confirms an user when created through the cli.
+
+#### Files integrity
+
+##### Checksum
+
+A task was added to automatically check whether files are corrupted, meaning that a file's checksum differs from the its original checksum.
+The celery task can be configured by editing the configuration `CELERY_BEAT_SCHEDULE['file-checks']`.
+
+##### Reports
+
+By default, everyday at 07:00 UTC a file integrity report is sent, by e-mail, in case a file is found to be corrupted.
+The celery task can be configured by editing the configuration `CELERY_BEAT_SCHEDULE['file-integrity-report']`.
+The e-mail fields can be modified by editing the following configurations:
+
+- `MAIL_DEFAULT_SENDER`: modifies the e-mail sender (field `from`).
+- `APP_RDM_ADMIN_EMAIL_RECIPIENT`: modifies the e-mail recipient (field `to`).
+- `FILES_INTEGRITY_REPORT_SUBJECT`: modifies the  subject of the e-mail (field `subject`).
+
+The e-mail template can be overriden completely by a custom one. To do so, edit the variable `FILES_INTEGRITY_REPORT_TEMPLATE` and point it to an alternative template.
+
+> Note: `MAIL_DEFAULT_SENDER` is a configuration used by `Flask-Mail`. If set, you don’t need to set the message sender explicity, as it will use this configuration value by default.
+
+#### URL redirections
+
+The redirector module was added to Invenio RDM. It allows an instance to add URL redirections using a configuration variable. See instructions on how to add an url redirection in its [howto](../develop/howtos/route_migration.md)
+
+#### Search query parser
+
+Search parameters may change overtime. It was added the possibility to map legacy search terms into newer terms. See instruction on how to add search terms mappings in its [howto](../develop/howtos/search_terms_migration.md)
+
+#### Default global visibility setting
+
+New users' profile and e-mail visibility can now be set by default to either `restricted` or `public`, by editing the configuration variables `ACCOUNTS_DEFAULT_USER_VISIBILITY` and `ACCOUNTS_DEFAULT_EMAIL_VISIBILITY` respectively. This change only affects new accounts, already existing accounts will keep their profile and e-mail visibility.
+
+#### Oauth
+
+TODO
+
+#### Custom views
+
+To extend your instance with your own custom views, you can use the predefined “site” folder in your instance. 
```

Comment:
I would suggest adding a little bit more context here, it is not very clear what's new here (at least for me :)):
```
You can now create your custom views or implement new features by adding your code in the new "site" folder in your instance.
```